code/modules/sanity/breakdowns.dm

```diff
@@ -432,6 +435,93 @@
 		holder.owner.stats.removeTempStat(stat, "Obsession")
 	..()
 
+
+/datum/breakdown/common/nostalgia
+	name = "Nostalgia"
+	restore_sanity_post = 40
+	var/area/nostalgic_target
+
+
+	start_messages = list(
+		"You feel that you've left something somewhere aboard. Something important...",
+		"There is a place where I understood my true self. I should get there again.",
+		"You suddenly feel that some place is pulling you towards itself slowly yet powerfully..."
+	)
+	end_messages = list(
+		"Finally, I am here."
+	)
+
+
+
+/datum/breakdown/common/nostalgia/New()
+	..()
+	nostalgic_target = pick(ship_areas)
+
+/datum/breakdown/common/nostalgia/update()
+	. = ..()
+	if (get_area(holder.owner) == nostalgic_target)
+		conclude()
+
+
+/datum/breakdown/common/high_as_hell
+	name = "High as Hell"
+	restore_sanity_post = 100
+
+	start_messages = list(
+		"You feel that you've left something somewhere aboard. Something important...",
+		"There is a place where I understood my true self. I should get there again.",
+		"You suddenly feel that some place is pulling you towards itself slowly yet powerfully..."
+	)
+	end_messages = list(
+		"OH FUCK.",
+		"FUCK YEAH",
+		"I need to rethink my life choices."
+	)
+
+
+/datum/breakdown/common/high_voltage
+	name = "High voltage"
+	restore_sanity_post = 100
+
+	start_messages = list(
+	"I need more power. Buzzing type."
+	)
+
+	end_messages = list(
+	"That's enough."
+	)
+
+/datum/breakdown/common/mechanicus
+	name ="Lust for Chrome"
+	restore_sanity_post = 80
+
+	start_messages = list(
+	"Flesh is weak, I need to replace it."
```

Comment:
"The flesh is weak, I need to replace it."
"I yearn for the strength of metal, over the weakness of the flesh."
"This body will not last long in its organic state, I must transcend it."

Reply:
Thanks for the suggestion, I didn't have much ideas about start/end messages.